app/controllers/admin/users_controller.rb

```diff
@@ -28,11 +28,19 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_user
-      @user = User.includes(:projects).find(params[:id])
+      @user = admin_user_policy_scope.includes(:projects).find(params[:id])
+      authorize @user
     end
 
     # Only allow a trusted parameter "white list" through.
     def user_params
       params.fetch(:user,{}).permit(:email, :name, :initials)
     end
+
+    def admin_user_policy_scope
+      # FIXME because it's a non-conventional policy class name, have to manually flag the verification
+      # is this the best way?
+      @_pundit_policy_scoped = true
```

Comment:
That's because I am using `after_filter :verify_authorized` in the application_controller. and here I am not using `policy_scope` so pundit will complain unless I set this variable (which is what policy_scope does)